src/Text2Image/T2IParamInput.cs

```diff
@@ -102,6 +103,8 @@ public class PromptTagContext
 
         public string[] Embeds, Loras;
 
+        public Dictionary<string, string> RandomVariables = [];
```

Comment:
just `Variables`, the vars themselves are not random

Reply:
Name left over from the original implementation, when it was just going to be for <random> and <wildcard>